### DISCUSSION TITLE
Extra tabs coming despite deleting them

### DISCUSSION THREAD

**Discussion by varunWinzo** [Budibase/budibase#4130] [Help]
![image](https://user-images.githubusercontent.com/97279160/150374141-b1eab217-9d4e-4950-809d-ff64272f42fe.png)

I create a sample budibase app to explore other things, but even after deleting that app, there names are still shown in the tabs. 
I have attached the screeshot of the same. Like in the image, I want to remove the names temporary and hj from the page. Please suggest how to remove this from my page.

**Comment by mjashanks** [comment]
Hi @varunWinzo you can configure this in [Layouts](https://docs.budibase.com/building-apps/design/layouts)
